tests/security/selinux/selinux_setup.pm

```diff
@@ -32,7 +32,17 @@ sub run {
     }
 
     # program 'sestatus' can be found in policycoreutils pkgs
-    zypper_call("in policycoreutils policycoreutils-python");
+    zypper_call("in policycoreutils");
+    my $ret = script_run('zypper -n in policycoreutils-python');
+    if ($ret == 104) {
```

Comment:
I think it is better to change to judgement to a more reasonable if switch. Such as:
if (!$ret) {
    if ($ret == 104) {}
    else {}
}

Reply:
done, thanks! and the run is good